common/src/main/java/tachyon/Constants.java

```diff
@@ -236,8 +236,8 @@
   public static final String MASTER_COLUMN_FILE_PREFIX = "COL_";
 
   public static final String LOGGER_TYPE = System.getProperty(TACHYON_LOGGER_TYPE, "");
-  public static final String ACCESS_LOGGER_TYPE =
-      System.getProperty(TACHYON_ACCESS_LOGGER_TYPE, "");
+  public static final String ACCESS_LOGGER_TYPE = System
+      .getProperty(TACHYON_ACCESS_LOGGER_TYPE, "");
```

Comment:
Surprising that the auto formatter prefers this, since it usually does it the other way. When I auto format it does this as well, perhaps its worth looking into why the style is inconsistent at times